samples/kotlin_compose/build.gradle.kts

```diff
@@ -18,6 +29,9 @@ android {
         vectorDrawables {
             useSupportLibrary = true
         }
+
+        buildConfigField("String", "API_KEY", "\"" + localProperties["apiKey"] + "\"")
+        buildConfigField("String", "SITE_ID", "\"" + localProperties["siteId"] + "\"")
```

Comment:
Nothing to take action in this PR, but just wanted to add that in future, we might be able to move out similar settings in a common gradle file and use `apply from` to add config fields and customer.io dependencies 💭

Reply:
This comment might come off a little bias, haha, but [here is a tool for us to consider](https://github.com/levibostian/dotenv-android) for providing the site id and api key to the app at compile time. 

I like the gradle plugin because we can create environment variables on the CI server and then all of our sample apps compile based on those environment variables. No need to worry about generating build config properties or `gradle.properties` files which are popular in Android projects.

Reply:
Thanks for the suggestions Levi, currently I am not sure how this solution is making our lives easier. Because the gradle and build config seems easier to me. 

All we need in the CI pipeline is to create a `local.properties` file in the root project with the keys as we do it in RH and a gradle script

```
buildConfigField("String", "API_KEY", "\"" + localProperties["apiKey"] + "\"")
buildConfigField("String", "SITE_ID", "\"" + localProperties["siteId"] + "\"")
``` 

that all sample apps `apply` from. And these keys would be available at compile time. 

Whereas the plugin you mentions, looks great but also needs a `file` to be used, and some other config and would provide the keys in compile time.

Reply:
Agreed with Shahroz here. Also, the changes I [pushed in java sample PR](https://github.com/customerio/customerio-android/pull/198/files#diff-82bfd67dff87b442354e11655db6249d10cb394b1981a4e4de5b2f3d0198a3db) makes it more easier. Once they are merged, I can pull the changes to separate gradle file use `apply from` for all our sample apps in the SDK. That will not only avoid repetition in gradle files, but will also help us setup CIO properties using the same idea that we have followed in RH and also making it simple for CI server.